src/main/js/util/configurationRequestBuilder.js

```diff
@@ -65,6 +65,7 @@ export function createUpdateRequest(apiUrl, csrfToken, configurationId, fieldMod
         method: 'PUT',
         body: JSON.stringify(fieldModel),
         headers: {
+            accept: 'application/json',
```

Comment:
Should usage of "accept" be quoted like the other headers?

Reply:
I'm not entirely sure, I was basing this on export functions just above this one where the accept: 'application/json' wasn't in quotes either

Reply:
RequestUtilities also seems to have the accept outside of quotes, so I think this may be intended

Reply:
Our linting rules declare the quotes as unnecessary for the accept header.

Reply:
We should probably refactor this code to use the Headers object and append the headers. I would probably write up an enhancement ticket for that.